x-pack/plugins/security_solution/public/common/containers/matrix_histogram/index.test.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/*
```

Comment:
it looks like the editor or linter inserted a duplicate header here

Reply:
Ahhh, how did I miss this??  Good catch Mr. Goldstein 🙂